python/ray/tests/test_job.py

```diff
@@ -56,14 +56,12 @@ def test_invalid_gcs_address():
         JobSubmissionClient("abc:abc")
 
 
-@pytest.mark.skipif(
-    sys.version_info >= (3, 12), reason="lib is not supported on Python 3.12"
```

Comment:
`lib` is the Python file we created for this test. I don't know why it says that `lib` is not supported on Python 3.12.

Reply:
lol